football-club-management/src/test/java/org/footballclub/footballclubmanagement/repository/StadiumRepositoryTest.java

```diff
@@ -0,0 +1,39 @@
+package org.footballclub.footballclubmanagement.repository;
+
+import lombok.extern.slf4j.Slf4j;
+import org.footballclub.footballclubmanagement.config.TestConfig;
+import org.footballclub.footballclubmanagement.model.StadiumEntity;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.transaction.annotation.Transactional;
+
+import static org.junit.jupiter.api.Assertions.*;
+@DataJpaTest
+@Import(TestConfig.class)
```

Comment:
use a dedicated application properties or yaml file for test profile rather than creating a test specific spring configuration bean so that the values are easily modifiable and transferrable